fulcrumapp/fulcrum-query-sql
# Bug/FLCRM-10130 Add Record Series SQL

Creates JOINs to include the RecordSeries data. 

[FLCRM-10130](https://fulcrumapp.atlassian.net/browse/FLCRM-10130) 

## Testing

This fixes the sorting and filters - it works in tandem with [this PR](https://github.com/fulcrumapp/fulcrum/pull/6893) that removes the breaking code.
Checkout this branch `bug/FLCRM-10130-fix-task-filter-sorting`

* In `fulcrum-components`
  * In your `package.json` file
    * change the fulcrum-query-sql version to this branch - it should look like:
    * `fulcrum-query-sql: github:fulcrumapp/fulcrum-query-sql#add-record-series-sql` 
  * Delete the existing `yarn.lock` file  
  * Rebuild the lock file by running `./yarn` 
  * Re-skaffold and now it's ready for testing
* Confirm that sorting and the filters are working
* Confirm that you can see the record series data come back from existing records
